app/addons/documents/shared-resources.js

```diff
@@ -27,7 +27,7 @@ define([
       return FauxtonAPI.constants.DOC_URLS.GENERAL;
     },
     url: function (context) {
-      if (context === undefined) {
+      if (!context) {
```

Comment:
who calls `Model.url(0);` and why? and if they do I guess they are happy with an url from the server